### DISCUSSION TITLE
Default tags for new recipes

### DISCUSSION THREAD

**Discussion by pndehoog** [mealie-recipes/mealie#4108] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Mealie is to be used by every person in the family. Not everyone is keen on categorizing and tagging recipes. To keep Mealie as clean as possible I need to identify newly created recipes (whether they are imported or manually created).

### Please provide a concise description of the feature that would resolve your issue.

In the group configuration you can set default settings for recipes. If a default tag was to be added, I could then create a tag that would identify a recipe as new on creation of such recipe.

After verification and cleansing the tag would be removed and the recipe would be approved.

### Please consider and list out some caveats or tradeoffs made in your design decision

I specifically request this feature for maintaining data quality. However, this feature can also be used for other purposes.  

The issue of data quality could also be solved by implementing a workflow in Mealie or setting a marker on a recipe. I can imagine that to be of greater impact on the overall design of Mealie.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by boc-the-git** [comment]
Have a read of this @pndehoog : https://github.com/mealie-recipes/mealie/pull/2930
